README.md

```diff
@@ -6,7 +6,7 @@
 With the appropriate parameters, run:
 
 ``` 
-$./autoracle -oracle_key_file="./keystore/key" -oracle_key_password="123" -oracle_autonity_ws_url="ws://127.0.0.1:8546" -oracle_plugin_conf="./plugins/plugin-conf.yml"
+$./autoracle -key.file="./keystore/key" -key.password="123" -autonity.ws.url="ws://127.0.0.1:8546" -plugin.conf="./plugins/plugin-conf.yml"
```

Comment:
I think it would be best to use the `--` prefix for long names and the `-` for shortcuts. That's the standard in linux cli tools AFAIK.

I checked and saw that the autoracle works also using the `--` prefix, it is just the documentation that uses the `-` prefix.

Reply:
Also maybe we can shorten the websocket option to `--ws`. Then we can specify in the description of the usage help that it is the websocket of autonity.

Reply:
Actually I saw that `go tool pprof` uses `-` even for long options as you did here in the oracle. I could be mistaken on this point, I am not sure which is the convention for golang programs. @Jason-Zhangxin-Chen  is it to use `-` ?